README.md

```diff
@@ -1,6 +1,8 @@
 # Fastai2 Audio
 > An audio module for v2 of fastai. We want to help you build audio machine learning applications while minimizing the need for audio domain expertise. Currently under development.
 
+# Quick Start
+[Google Colab Notebook](https://colab.research.google.com/gist/PranY/ba0245752fff8ec2eb645afcc13f74f6/music.ipynb)
```

Comment:
Yes, I realized that I edited the markdown instead of the notebook and any changes in the notebook would override it later. So I edited the notebook and added the video link as well and built from the notebooks.

In theory, that's the same as the previous PR but it overrides the previous PR in a way which is consistent with the development and test style of the library. Does that make sense? Will this be a merge conflict?